infra/luci/generated/luci-scheduler.cfg

```diff
@@ -53,7 +53,7 @@ trigger {
   triggers: "perfetto-official-builder-mac"
   triggers: "perfetto-official-builder-windows"
   gitiles {
-    repo: "https://android.googlesource.com/platform/external/perfetto"
+    repo: "https://github.com/google/perfetto"
```

Comment:
does this work? I thought we had to use the chromium external/github.com mirror as LUCI has restricted network?

Reply:
Fixed.